mappings/net/minecraft/block/TrialSpawnerBlock.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_qiypfnai net/minecraft/block/TrialSpawnerBlock
+	FIELD f_eqxijsod STATE Lnet/minecraft/unmapped/C_cgckxfsw;
+	METHOD m_hqexpyll (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_ndgqjckh;)V
+		ARG 1 world2
```

Comment:
world2?

Reply:
It's a lambda, `world` exists in the outer method, and I didn't know what to name this one (it isn't used anyway)

Reply:
How about unused?

Reply:
`unusedWorld` or just `unused`?

Reply:
`unusedWorld` imo

Reply:
Fixed, btw